tests/fixes/fixtures/complicated.env

```diff
@@ -70,4 +70,16 @@ VAR_4=1234
 VAR_3=1234
 VAR_2=1234
 VAR_1=1234
-ENV24='key56'
\ No newline at end of file
+ENV24='key56'
+# dotenv-linter:on QuoteCharacter
+
+# multiline values & ordering:
```

Comment:
This block shows that it works as expected: values will be trimmed and reordered